app/src/main/java/com/eveningoutpost/dexdrip/NewSensorLocation.java

```diff
@@ -0,0 +1,162 @@
+package com.eveningoutpost.dexdrip;
+
+import android.app.Activity;
+import android.content.Intent;
+import android.content.SharedPreferences;
+import android.os.Bundle;
+import android.preference.PreferenceManager;
+import android.support.v4.widget.DrawerLayout;
+import android.util.Log;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.view.View;
+import android.widget.Button;
+import android.widget.CheckBox;
+import android.widget.DatePicker;
+import android.widget.EditText;
+import android.widget.LinearLayout;
+import android.widget.RadioButton;
+import android.widget.RadioGroup;
+import android.widget.RadioGroup.OnCheckedChangeListener;
+import android.widget.TimePicker;
+import android.widget.Toast;
+
+import com.eveningoutpost.dexdrip.UtilityModels.CollectionServiceStarter;
+import com.eveningoutpost.dexdrip.utils.ActivityWithMenu;
+
+import java.util.Calendar;
+import java.util.LinkedList;
+import java.util.List;
+
+class Location {
+    Location(String location, int location_id) {
+        this.location = location;
+        this.location_id = location_id;
+    }
+    public String location;
+    public int location_id;
+}
+
+
+public class NewSensorLocation extends ActivityWithMenu {
+    public static String menu_name = "New sensor location";
+    private Button button;
+    private Button buttonCancel;
+    private RadioGroup radioGroup;
+    private EditText sensor_location_other;
+    CheckBox DontAskAgain;
+    List<Location> locations;
+    
+    final int PRIVATE_ID = 200;
+    final int OTHER_ID = 201;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_new_sensor_location);
+        button = (Button)findViewById(R.id.saveSensorLocation);
+        buttonCancel = (Button)findViewById(R.id.saveSensorLocationCancel);
+        sensor_location_other = (EditText) findViewById(R.id.edit_sensor_location);
+        sensor_location_other.setEnabled(false);
+        DontAskAgain = (CheckBox)findViewById(R.id.sensorLocationDontAskAgain);
+        radioGroup = (RadioGroup) findViewById(R.id.myRadioGroup);
+        addListenerOnButton();
+        
+        locations = new LinkedList<Location>();
+        
+        locations.add(new Location("I dont wish to share", PRIVATE_ID));
```

Comment:
Can you change this to either "I do not wish to share" or "I don't wish to share"?